e2e/cypress/integration/integrations/slash_commands_spec.js

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+/**
+* Note: This test requires webhook server running. Initiate `npm run start:webhook` to start.
+*/
+
+describe('Integrations', () => {
+    let user1;
+    let user2;
+    let team1;
+    let team2;
+    let testChannelUrl1;
+    let testChannelUrl2;
+    let commandURL;
+
+    const commandTrigger = 'test-ephemeral';
+    const timestamp = Date.now();
+
+    before(() => {
+        cy.requireWebhookServer();
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                EnableLinkPreviews: true,
+            },
+        });
+
+        cy.apiInitSetup().then(({team, user}) => {
+            user1 = user;
+            team1 = team;
+            testChannelUrl1 = `/${team.name}/channels/town-square`;
+            cy.apiGetChannelByName(team1.name, 'town-square').then((response) => {
+                const channel = response.body;
```

Comment:
Sorry about this but I recently updated [this command](https://github.com/mattermost/mattermost-webapp/pull/6472/files#diff-f818c44918beda35ee35b64faf605829R105-R113) for consistency. Please change to
```suggestion
            cy.apiGetChannelByName(team1.name, 'town-square').then(({channel}) => {
```